CoseIndirectSignature/IndirectSignatureFactory.Bytes.cs

```diff
@@ -188,20 +188,19 @@ public CoseSign1Message CreateIndirectSignatureFromHash(
     /// <param name="coseHeaderExtender">Optional header extender to add custom headers to the COSE message.</param>
     /// <returns>A Task which can be awaited which will return a CoseSign1Message which can be used as a Indirect signature validation of the payload.</returns>
     /// <exception cref="ArgumentNullException">The contentType parameter was empty or null</exception>
-    public Task<CoseSign1Message> CreateIndirectSignatureAsync(

```

Comment:
All async methods accept a CancellationToken cancellationToken = default parameter internally but don't expose it in the public async API surface. This means callers cannot cancel long-running async operations.

All CreateIndirectSignatureAsync variants (lines 191, 216, 189, 214 across Bytes.cs and Stream.cs)
All CreateIndirectSignatureBytesAsync variants (lines 413, 438, 411, 437)
All CreateIndirectSignatureFromHashAsync variants